### DISCUSSION TITLE
Q: Svelte support

### DISCUSSION THREAD

**Discussion by nea** [coreui/coreui#523] [General]
Hi there

Do you intend to support Svelte with a native template and component pack in the future?

Thanks a lot

**Comment by mrholek** [comment]
Hi @nea, 

I can't promise anything at this moment, but we're thinking about it.

**Comment by nea** [comment]
Hey @mrholek 

That is already fine for me ^^ As long as you are thinking about it there will be hope ;)


Thanks a lot

**Comment by baradhili** [comment]
with svelte maturing quite fast - bumping this

**Comment by baradhili** [comment]
so while I agree - there are only so many frameworks that can be maintained, I'm wondering on people's feeling between Svelte and Vue.. they seem fairly similar in coding style - though obviously there are big differences

**Comment by baradhili** [comment]
This might be a way to deal with this.. https://github.com/builderio/mitosis

**Comment by mrholek** [comment]
@baradhili it's all about time and money, at this moment we don't have free resources to maintain another framework, so the only way is to hire more developers and invest some money to create it.
